readme.md

```diff
@@ -19,25 +18,27 @@ This project will assess the following learning objectives, in addition to many
 Follow the instructions below to get this project up and running on your own machine:
 
 - Fork and clone this repository.
-- Run `npm install`.
+- Run `npm install` to install the dependencies needed for this project.
```

Comment:
since this exercise is pretty early on in the curriculum, might be helpful to remind students why they are running certain commands even if they have seen them before.